### DISCUSSION TITLE
Replace organisation prefix (oid_prefix)

### DISCUSSION THREAD

**Discussion by TriformMoritz** [QGEP/QGEP#691] [Q&A] [answered]
Hello,

my data was imported without the organization prefix  (oid_prefix), so all the automatic prefixes in the postgres database are "00000000".

_Is there an **easy way** to replace all the prefixes in the database with the official Interlis prefix?_ 

Idea (not sure if it works): Inside postgres, disable the dependencies temporarily and then replace all oid values, regenerate dependencies,

**Comment by TriformMoritz** [comment, marked as answer]
Somebody could help me? I got the new prefix but I do not know how to replace the prefix in all TID / OID.
Thank you

**Reply by urskaufmann** [reply]
I would export in INTERLIS, replace in the XTF File and reimport the data

**Comment by TriformMoritz** [comment]
Thank you Urs for your advice. 
Export is not possible, because I do not have pip3 installed and I do not know how to install it.
Machine is on windows 10. OSGeo4W Shell seems to work with python 2 (even if 3 is installed):

C:\>python -c "import sys; print(sys.executable)"
C:\OSGEO4~1\bin\python.exe

For python3 I got the following error:

run o-help for a list of available commands
C:\>python3
Fatal Python error: initfsencoding: unable to load the file system codec
  File "C:\OSGEO4~1\apps\Python27\lib\encodings\__init__.py", line 123
    raise CodecRegistryError,\
                            ^
SyntaxError: invalid syntax

Current thread 0x00002bac (most recent call first):
